src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -262,14 +257,25 @@ export default {
     }
 
     if (this.mutablePromise) {
-      this.mutablePromise.then(() => {
-        this.openmct.objects.destroyMutable(this.domainObject);
-      });
+      await this.mutablePromise();
+      this.openmct.objects.destroyMutable(this.domainObject);
     } else if (this?.domainObject?.isMutable) {
       this.openmct.objects.destroyMutable(this.domainObject);
     }
   },
   methods: {
+    async getAndSetObject() {
+      let foundObject = null;
+      if (this.openmct.objects.supportsMutation(this.item.identifier)) {
+        this.mutablePromise = this.openmct.objects.getMutable(this.item.identifier);
+        foundObject = await this.mutablePromise;
+      } else {
+        foundObject = await this.openmct.objects.get(this.item.identifier);
+      }
```

Comment:
@scottbell any idea why this isn't being called in our tests?

Reply:
I think because the objects we're dealing with support mutation.